src/persist/src/indexed/arrangement.rs

```diff
@@ -1276,14 +1269,6 @@ mod tests {
         assert_eq!(t.validate_allow_compaction(&Antichain::from_elem(5)),
             Err("invalid compaction less than trace since Antichain { elements: [6] }: Antichain { elements: [5] }".into()));
 
-        // Advance since frontier to seal
```

Comment:
let's keep these tests but now verify that they succeed

Reply:
:+1: